mitodl/micromasters
# Added migration to set ref numbers on old orders

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #5213 

#### What's this PR do?
This PR adds a migration to set the reference number on historical orders and a small refactor of a constant (existing tests cover this).

#### How should this be manually tested?
- Ensure you have old orders with `reference_number=None`
  - You can run `Order.objects.filter(reference_number=None).exists()` to verify this quickly
- Run the migration
- Verify you now see reference numbers in django-admin
- Run `Order.objects.filter(reference_number=None).exists()` again to verify no more orders exist with null `reference_number` values.
